perfkitbenchmarker/providers/aws/aws_network.py

```diff
@@ -51,7 +51,7 @@ def __init__(self):
     self.firewall_set = set()
     self._lock = threading.Lock()
 
-  def AllowPort(self, vm, port):
+  def AllowPort(self, vm, port, to_port=None):
```

Comment:
When I first saw the name `to_port`, I thought this was a firewall rule targeting connections with different source and dest ports. Maybe rename `port` to `start_port` and then `to_port` to `end_port`, or something like that?